h2o-docs/src/product/automl.rst

```diff
@@ -265,13 +259,13 @@ FAQ
 
 -  **Which models are trained in the AutoML process?**
 
-  The current version of AutoML trains and cross-validates a default Random Forest, an Extremely-Randomized Forest, a random grid of Gradient Boosting Machines (GBMs), a random grid of Deep Neural Nets, a fixed grid of GLMs, and then trains two Stacked Ensemble models.  A list of the hyperparameters searched over for each algorithm in the AutoML process is included in the appendix below.
+  The current version of AutoML trains and cross-validates a default Random Forest, an Extremely-Randomized Forest, a random grid of Gradient Boosting Machines (GBMs), a random grid of Deep Neural Nets, a fixed grid of GLMs, and then trains two Stacked Ensemble models.  A list of the hyperparameters searched over for each algorithm in the AutoML process is included in the appendix below.  More details about the hyperparamter ranges for the models will be added to the appendix at a later date.
```

Comment:
I still think the GLM is not technically a grid. More a search over alpha and lambda but not a grid in the sense we get back several models (just one).

Reply:
You're right, it's not technically an "H2O Grid" object in the code, but it's a grid in theory (we train a grid over alphas, but only return the top model), so I think for now it's still okay to say it "trains a grid of GLMs" since that part is still true.

Reply:
Yes, maybe we can add a note about what this means for the GLM case? i.e., its not an `H2O Grid` so there will only be one model (the best model).